File: C#/beecrowd-1005.cs
using System; 

class URI {
    const double PESO_TOTAL = 11.0;
    const double PESO_A = 3.5;
    const double PESO_B = 7.5;
    
    static void Main(string[] args) { 
        double notaA;
        double notaB;
        double media;

        notaA = double.Parse(Console.ReadLine());
        notaB = double.Parse(Console.ReadLine());
        
        media = (notaA * PESO_A + notaB * PESO_B) / PESO_TOTAL;

        Console.WriteLine(String.Format("MEDIA = {0:0.00000}", media));
    }
}

File: Elixir/beecrowd-1005.exs
# Input
a = IO.gets("") |> String.trim |> String.to_float
b = IO.gets("") |> String.trim |> String.to_float

pesoA = 3.5
pesoB = 7.5
pesoTotal = 11.0

# Processing
result = ((a * pesoA) + (b * pesoB)) / pesoTotal
result = if result > 10.0, do: 10.0, else: result
result = :erlang.float_to_binary(result, [decimals: 5])

# Output
IO.puts("MEDIA = #{result}")
